src/foam/nanos/column/ColumnConfigPropertyValue.js

```diff
@@ -64,33 +64,37 @@ foam.CLASS({
       code: function(of, propName) {
         var cls = of;
         var axiom;
-        var columnHeader = [];
+        var colPath = []; // Array with full path to column header
+        var colLabel = ''; // Returns columnLabel property if applicable
 
```

Comment:
returnColumnHeader used in couple of different places for example in `src/foam/nanos/export/GoogleSheetsOutputter.js`.

so we'll need to do            ` columnLabel: columnConfig.returnColumnHeader(of, propName)[1],` on line 89`src/foam/nanos/export/GoogleSheetsOutputter.js`.